packages/react-swipeable-views/src/SwipeableViews.js

```diff
@@ -746,6 +756,17 @@ class SwipeableViews extends Component {
     const touchEvents = disabled ? {} : {
       onTouchStart: this.handleTouchStart,
       onTouchEnd: this.handleTouchEnd,
+      onMouseDown: (e) => {
+        e.preventDefault();
```

Comment:
In my case, I had an image in the panel that has a default drag behaviour. This was to prevent that from happening.